watchdog/watchdog.go

```diff
@@ -26,9 +26,11 @@ var (
 func Monitor(cfg *config.Config) {
 	ctx, cancelFunc = context.WithCancel(context.Background())
 	for _, service := range cfg.Services {
-		// To prevent multiple requests from running at the same time, we'll wait for a little bit before each iteration
-		time.Sleep(1111 * time.Millisecond)
-		go monitor(service, cfg.Alerting, cfg.DisableMonitoringLock, cfg.Metrics, cfg.Debug, ctx)
+		if *service.Enabled {
```

Comment:
Could you add a method `IsEnabled` instead and use that?

For an example, see https://github.com/TwinProduction/gatus/blob/18420c2d60d1190510f7e02661102b4bc7e0c2ac/alerting/alert/alert.go#L56

Call me paranoid; I'm aware that `ValidateAndSetDefaults` will be called before this, but I'd rather do this to avoid a possible panic due to a code change in the future.

Reply:
Thank you for your review. Fixed in 8d4b3d3.